src/main/java/by/exadel/internship/service/impl/InternshipServiceImpl.java

```diff
@@ -131,6 +135,18 @@ public UserInternshipDTO saveInternship(UserInternshipDTO internshipDTO) {
         return userInternshipDTO;
     }
 
+    public boolean addUser(UUID userId, UUID internshipId) {
+        internshipRepository.getOne(internshipId);
```

Comment:
why are you getting it if not using later?

Reply:
remove